### PR TITLE
Navigation Layout Fragrments

### PR DESCRIPTION
**Pull request by keyoni**
All main fragments are created and connected to proper places via nav_main_app.xml

### CODE REVIEW

**Review by BorisMarin8004 — APPROVED** [review]

**Comment by BorisMarin8004** [comment]
Looks great!

File: Android/app/src/main/res/navigation/nav_main_app.xml [hunk 1]
```diff
@@ -13,13 +13,25 @@
 
         <action
             android:id="@+id/action_FirstFragment_to_SecondFragment"
```
Comment (bKondo) [inline Android/app/src/main/res/navigation/nav_main_app.xml:15]: are we going to rename this to `action_HomePostFragment_to_eventsFragment`?

**Review by bKondo — APPROVED** [review]
nice, we have the base pieces connected now

**Review by parw8649 — APPROVED** [review]

File: Android/app/src/main/res/navigation/nav_main_app.xml (hunk 1 above)
Comment (keyoni) [reply Android/app/src/main/res/navigation/nav_main_app.xml:15]: Yeah, sorry about that!

**Pull request merged**